tests/test_objective_funs.py

```diff
@@ -1292,8 +1292,8 @@ def test_derivative_modes():
     )
     obj2 = ObjectiveFunction(
         [
-            PlasmaVesselDistance(eq, surf),
```

Comment:
Can we add a test that check chunked and normal batched gives the same result?

Reply:
batched always will use chunked now, since `None` defaults to `dim_x/4`